src/avfoundation.cs

```diff
@@ -4081,7 +4082,8 @@ interface AVAssetWriter {
 		[TV (14, 0), NoWatch, Mac (11, 0), iOS (14, 0)]
 		[Export ("outputFileTypeProfile")]
 		[NullAllowed]
-		string OutputFileTypeProfile { get; set; }
+		[BindAs (typeof (AVFileTypeProfile))]
+		NSString OutputFileTypeProfile { get; set; }
```

Comment:
missed it earlier but there's a `[NullAllowed]` here
so the smart enum should have a default member, like
```
[Field (null)]
None = 0,
```
Ideally at first position (0) when possible. Otherwise you won't be able to set the property to `null` anymore